radxa/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+let
+  cfg = config.hardware.radxa;
+in {
+  options.hardware.radxa = {
+    enable = lib.mkEnableOption "Radxa system support";
+  };
+
+  config = lib.mkIf cfg.enable {
+    boot = {
+      # Currently enable bcachefs automatically set
+      # kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
+      kernelPackages = lib.mkOverride 990 pkgs.linuxPackages_latest;
```

Comment:
```suggestion
      kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "6.12") pkgs.linuxPackages_latest;
```

Reply:
I think `kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;` is still better. The intention is to always use the latest kernel so newer features can be available sooner (like RK3588 display in 6.13), and `bcachefs` is enabled because we are using the latest kernel.

With this on newer nixpkgs the default kernel will be 6.12 until next LTS.